developer-docs/bolt_server.md

```diff
@@ -11,10 +11,13 @@ Bolt server can be configured by defining content in HOCON format at one of the
 **Local Config**: `~/.puppetlabs/bolt-server.conf`
 
 **Options**
-- `port`: Integer, *optional* - The port the bolt server will run on (default 62658)
+- `host`: String, *optional* - Hostname for server (default "127.0.0.1").
+- `port`: Integer, *optional* - The port the bolt server will run on (default 62658).
 - `ssl-cert`: String, *required* - Path to the cert file.
 - `ssl-key`: String, *required* - Path to the key file.
 - `ssl-ca-cert`: String, *required* - Path to the ca cert file.
+- `loglevel`: String, *optional* - Bolt log level, acceptable values are `debug`, `verbose`, `notice`, `warn` (default `notice`).
```

Comment:
These should be debug, info, notice, warn, error.

Reply:
@katelopresti, The pre-docs were updated in #595. The bolt docs on puppet.com shows the "old" values. https://puppet.com/docs/bolt/0.x/bolt_configuration_options.html#log-file-configuration-options. I see this was a very recent change. Just wanted to comment here. Should have checked the source code for this.